deploy/olm-catalog/performance-addon-operator/performance-addon-operator.package.yaml

```diff
@@ -1,7 +1,7 @@
 channels:
 - currentCSV: performance-addon-operator.v4.4.0
-  name: 4.4.0
-- currentCSV: performance-addon-operator.v4.5.0
-  name: 4.5.0
-defaultChannel: 4.5.0
+  name: "4.4"
```

Comment:
I guess this can be surprising for people still on 4.4, but makes sense in the big picture, so I'm for it. @MarSik any comments?

Reply:
ok, Martin already gave them :)